File: CourseProjects/CourseProjects/VisualStudioProjects/Agile/IntroducingNUnit/Backup/NUnitWithVB/BasicMath.vb
Public Class BasicMath
    Public Sub New(ByVal no1 As Integer, ByVal no2 As Integer)
        Me.no1 = no1
        Me.no2 = no2
    End Sub
    Public Function add() As Integer
        Return no1 + no2
    End Function
    Public Function subtract() As Integer
        Return no1 - no2
    End Function
    Public Function multiply() As Integer
        Return no1 * no2
    End Function
    Private no1 As Integer
    Private no2 As Integer
End Class

File: CourseProjects/CourseProjects/VisualStudioProjects/Agile/IntroducingNUnit/NUnitWithCS/BasicMath.cs
using System;

namespace TestBasicMath {
	public class BasicMath {
		public BasicMath(int no1, int no2) : base() {
			this.no1 = no1;
			this.no2 = no2;
		}
		public int Add() {
			return no1 + no2;
		}
		public int Subtract() {
			return no1 - no2;
		}
		public int Multiply() {
			return no1*no2;
		}
		private int no1;
		private int no2;
	}
}